lib/capybara/node/actions.rb

```diff
@@ -6,36 +6,32 @@ module Actions
       # @!macro waiting_behavior
       #   If the driver is capable of executing JavaScript, this method will wait for a set amount of time
       #   and continuously retry finding the element until either the element is found or the time
-      #   expires. The length of time +find+ will wait is controlled through {Capybara.default_max_wait_time}
+      #   expires. The length of time {Capybara::Node::Finders#find} will wait is controlled through {Capybara.configure default_max_wait_time}.
```

Comment:
This is a macro that gets applied to other methods - please just change `+find+` to `this method` - I think $0 is supposed to reference to the method the macro is applied to but after a quick (but limited test) it doesn't appear to work in this context - not really sure why

Reply:
`The length of time {$0} will wait` also works well, but I also think just `this method` is good here. I'll fix it. 👍 

![image](https://user-images.githubusercontent.com/473530/58059940-07618880-7baa-11e9-96e5-d7b79e667579.png)